enterprise/admin/forms.py

```diff
@@ -65,13 +71,25 @@ class Fields(object):
         MODE = "mode"
         COURSE = "course"
         COURSE_MODE = "course_mode"
+        PROGRAM = "program"
 
     class CsvColumns(object):
         """
         Namespace class for CSV column names.
         """
         EMAIL = "email"
 
+    def __init__(self, *args, **kwargs):
+        """
+        Initializes
```

Comment:
@mattdrayer sure. Looks like I've started typing a docstring, got distracted and forgot about it :)